dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -188,15 +188,25 @@ Future<void> _testBuildIosFramework(Directory projectDir, { bool isModule = fals
       'vm_snapshot_data',
     ));
 
+    final String dsymPath = path.join(
+        outputPath,
+        mode,
+        'App.xcframework',
+        'ios-arm64',
+        'dSYMs'
+    );
+    checkDirectoryExists(dsymPath);
+
     final String appFrameworkDsymPath = path.join(
-      outputPath,
-      mode,
-      'App.xcframework',
-      'ios-arm64',
-      'dSYMs',
-      'App.framework.dSYM'
+        dsymPath,
+        'App.framework.dSYM'
     );
     checkDirectoryExists(appFrameworkDsymPath);
+
+    if (Directory(dsymPath).listSync().length != 1) {
```

Comment:
Can you filter on directories, I'm afraid this will flake if a `.DS_Store` file sneaks in. Something like:
```suggestion
    if (Directory(dsymPath).listSync().whereType<Directory>().length != 1) {
```

Reply:
oh fair...

Reply:
changed it to prevent sneaky files